### ISSUE TITLE
Application fails to start

### ISSUE THREAD

**Issue by dylanzwick** [spring-guides/gs-accessing-data-mysql#6]
Run logs attached
[RunLogs.txt](https://github.com/spring-guides/gs-accessing-data-mysql/files/1254153/RunLogs.txt)

**Comment by dsyer** [comment]
Works for me. In your original issue (https://github.com/spring-guides/getting-started-guides/issues/51) you said your imported the project into eclipse. How did you do that? Did you try running in the command line? Clear your maven cache?

**Comment by dylanzwick** [comment]
Imported the project to Eclipse:
New -> Other -> Spring Boot / Import Getting Started Content -> Accessing Data MySQL
I also tried installing it directly from the attached file.
[gs-accessing-data-mysql-master.zip](https://github.com/spring-guides/gs-accessing-data-mysql/files/1254191/gs-accessing-data-mysql-master.zip)

I just ran in on the command line. Got the same error. The RunLogs are attached.
[RunLogs_2.txt](https://github.com/spring-guides/gs-accessing-data-mysql/files/1254192/RunLogs_2.txt)

I'm new to this, so I had to look up how to clear the maven cache, but in an attempt to do so I ran:
mvn dependency:purge-local-repository -DactTransitively=false -DreResolve=false --fail-at-end

This generated some warnings towards the end. The complete output is attached.
[RunLogs_3.txt](https://github.com/spring-guides/gs-accessing-data-mysql/files/1254194/RunLogs_3.txt)

I'll investigate those warnings, but so far I'm unable to run the application.
